src/lib/constants.js

```diff
@@ -215,6 +215,13 @@ const STAKEWISE_API_URLS = {
 const ZKSWAP_URL =
   'https://zkswap-dex-api-git-production-buzzteam.vercel.app/api/pools-data/zksyncMainnet'
 
+const SYNCSWAP_API_URL =
+  'https://api.syncswap.xyz/api/fetchers/fetchAllPools?network=zkSyncMainnet&type=v2'
+
+const SYNCSWAP_FEE_API_URL = 'https://api.syncswap.xyz/api/fetchers/fetchFees?network=zkSyncMainnet'
+
+const SYNCSWAP_SUBGRAPH_URL = 'https://api.studio.thegraph.com/query/30365/zksync-blocks/v0.0.9'
```

Comment:
Will this version keep up to date? Can we use `version/latest`?

Reply:
Maybe they can update the api and graph url in the future so we need to track them in time.